### PR TITLE
fix: remove Hong Kong from choosing-the-state-structure

### PR DESCRIPTION
**Pull request by umairraza96**
closes Improper examples issue number: #5955

### CODE REVIEW

File: src/content/learn/choosing-the-state-structure.md [hunk 1]
```diff
@@ -833,11 +829,12 @@ import { initialTravelPlan } from './places.js';
...
   const place = placesById[id];
   const childIds = place.childIds;
+   const isChildIdPresent = childIds && childIds.length > 0 
   return (
     <li>
       {place.title}
-      {childIds.length > 0 && (
-        <ol>
+      {
+      isChildIdPresent && <ol>
```
Comment (gaearon) [inline src/content/learn/choosing-the-state-structure.md:837]: This is an unrelated stylistic change? Let's not do that.

Comment (umairraza96) [reply src/content/learn/choosing-the-state-structure.md]: 👍

**Comment by mohammad-taheri1** [comment]
Instead of Changing everything, Just replace Hong Kong with another country Bro 😃

**Review by mohammad-taheri1 — COMMENTED** [review]
It seems Ok.
But, You could simply rename `Hong Kong` to another country like `Iran`

**Comment by rickhanlonii** [comment]
Thanks!
